feature-pack/src/main/resources/modules/system/layers/base/org/jboss/weld/core/main/module.xml

```diff
@@ -30,6 +30,9 @@
 
     <resources>
         <artifact name="${org.jboss.weld:weld-core-impl}"/>
+        <artifact name="${org.jboss.weld.module:weld-ejb}"/>
```

Comment:
Are there plans to move these into their own module so they can be excluded if not required (e.g. by swarm)?

Reply:
Good point. We'll create a separate ticket to address this issue.